kivy/resources.py

```diff
@@ -42,23 +43,41 @@
     resource_paths += [join(dirname(sys.argv[0]), 'YourApp')]
 resource_paths += [dirname(kivy.__file__), join(kivy_data_dir, '..')]
 
+Cache.register('kv.resourcefind', timeout=60)
 
-def resource_find(filename):
+
+def resource_find(filename, use_cache=True):
     '''Search for a resource in the list of paths.
     Use resource_add_path to add a custom path to the search.
+    By default, results are cached for 60 seconds.
+    This can be disabled using use_cache=False.
+    .. versionchanged:: 2.0.0rc5
```

Comment:
```suggestion
    .. versionchanged:: 2.0.0
```

Reply:
Just a reminder that this should not be merged in 2.0.0 as we're in a freeze until the release. Especially something with significant changes as this.

Reply:
The current version is now `2.1.0`.

Reply:
Fixed